bin/distmap

```diff
@@ -807,7 +801,8 @@ To run a test for all dependency and all.
 
 =item B<--quality-threshold>
 
-minimum average quality; A modified Mott algorithm is used for trimming; the threshold is used for calculating a score: score = quality_at_base - threshold; default=20
+OBSOLETE: minimum average quality; A modified Mott algorithm is used for trimming; the threshold is used for calculating a score: score = quality_at_base - threshold; default=20
```

Comment:
Arguments that still work as intended, but should not be used, should be marked "DEPRECATED". OBSOLETE" are arguments that do nothing useful anymore.

Reply:
Ok, I'll change tomorrow...

Reply:
Changed to DEPRECATED.